### ISSUE TITLE
Configuration Image docker

### ISSUE THREAD

**Issue by adelbot** [VictoriaMetrics/VictoriaMetrics#4907]
### Is your question request related to a specific component?

All products

### Describe the question in detail

In image for v1.93.0, the manifest indicate : "mediaType": "application/vnd.docker.distribution.manifest.v2+json"

In image for v1.93.1, the manifest indicate : "mediaType": "application/vnd.oci.image.manifest.v1+json"

But in CENTOS 7, docker version is docker-1.13.1 and is incompatible with **oci**.

### Troubleshooting docs

- [ ] General - https://docs.victoriametrics.com/Troubleshooting.html
- [ ] vmagent - https://docs.victoriametrics.com/vmagent.html#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert.html#troubleshooting

**Comment by f41gh7** [comment]
@adelbot  can you try `v1.93.2-rc1` images? I've build it on top of `v1.93.1` for community version of VictoriaMetrics.

**Comment by f41gh7** [comment]
docker output seems to be correct:
```
 docker manifest inspect victoriametrics/victoria-metrics:v1.93.2-rc1
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 899,
         "digest": "sha256:0a0c5e183ec4887e20e9e972f96d4ae06d6338a6e2c51e20c4625089872e99d5",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
```

**Comment by hagen1778** [comment]
@adelbot the fix to the docker buildx was already merged and will be used for building the next release. Are you willing to wait for the next release or it is crucial for you to get updated asap to 1.93.1?

**Comment by valyala** [comment]
The issue with non-working Docker images for older Docker releases has been fixed since [v1.93.2](https://docs.victoriametrics.com/CHANGELOG.html). Closing it as fixed.
